src/main/java/com/google/sps/data/SmallCityService.java

```diff
@@ -62,6 +80,6 @@ private void getSmallBusinesses() {
     businesses = new LinkedList<Listing>();
     businessesService = new BusinessesService(businesses);
     findAllBusinesses();
-    eliminateBigBusinesses();
+    parseThroughTheListOfBusinesses();
```

Comment:
Was this recently renamed? This method does actually do any parsing. Nonetheless it's still an odd naming convention. It looks, to me, like what's really happening is some kind of filtering. A list of businesses (small and big) is returned from the db. Then the bigBussinesses are being removed, and what's left is returned.

Reply:
I changed the method name into filterTheListOfBusinessesIntoSmallBusinesses instead hopefully this is better